FB.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.common.exceptions import TimeoutException
+import time
+class SignUpTest(unittest.TestCase):
+    def setUp(self):
+        self.driver= webdriver.Firefox(executable_path="/home/ruttaba/PycharmProjects/geckodriver")
+        self.driver.get("https://www.facebook.com/")
+        self.assertIn("Facebook", self.driver.title)
+        self.driver.maximize_window()
+    def test_enter_credentials_in_signUp_form(self):
+        #review test fb git
```

Comment:
@ruttaba Why this comment?

Reply:
because of the instructions given by Atia min Allah for an experiment